### ISSUE TITLE
Unknown error

### ISSUE THREAD

**Issue by kr812** [z-huang/music#1770]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions/workflows/build.yml).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Open > Play music (any music)> then you see error

### Expected behavior

Due to YouTube api updated

### Actual behavior

When I play any music on Innertube then I got this error see screenshot

### Screenshots/Screen recordings

![Screenshot_2024-12-22-13-09-14-313_com zionhuang music](https://github.com/user-attachments/assets/da051d31-b55e-4c18-8a92-1696d513e8f7)

### Logs

Sorry

### InnerTune version

0.5.10

### Android version

Android 14

### Additional information

But the same as your type one application name outertune is working good i think they update his own api you have too see outertune one time

**Comment by Santhanam1** [comment]
Have the same error

**Comment by kr812** [comment]
I reported Team let's see what's happened

**Comment by Rimuru171006** [comment]
I'm Having Same Problem

**Comment by prinstonedkhar** [comment]
Same issue here. Android 11.
Tbh innertube is the best from all the open-source music that I have tried.

**Comment by CrebsTheCoder** [comment]
Same issue

**Comment by 197ritesh** [comment]
Anyone able to figure a way out of this ??

**Comment by Santhanam1** [comment]
I installed outertube(a fork of innertune) they fixed that for now, I still not logined, working without login for now

**Comment by 197ritesh** [comment]
Yes outer tune is working fine you can import your playlist to that but I really want inner Tune to be fixed asap

**Comment by scindo07** [comment]
when fix

**Comment by GH1858** [comment]
Same as me.

Please fix as soon as possible.

**Comment by Mateus109** [comment]
Perhaps Google has finally blocked their YouTube API feeding InnerTune?

**Comment by Dmutro85** [comment]
Same issue :(
Android 12
App ve. 0.5.10 foss

**Comment by GH1858** [comment]
> Perhaps Google has finally blocked their YouTube API feeding InnerTune?

I don't know, but if they blocked their YouTube API, then I have to switch to OuterTune.

**Comment by 666niuniu999** [comment]
Same problem.🥲

**Comment by GH1858** [comment]
Hey z-huang, could you please and your team fix up the app?

**Comment by SonoPeppeDEV** [comment]
Same issue

**Comment by kr812** [comment]
> Hey z-huang, could you please and your team fix up the app?

I found the solution just logout your account

**Comment by kr812** [comment]
> Same issue

I found the solution just logout your account

**Comment by kr812** [comment]
> > Perhaps Google has finally blocked their YouTube API feeding InnerTune?
> 
> I don't know, but if they blocked their YouTube API, then I have to switch to OuterTune.

I found the solution just logout your account

**Comment by Mateus109** [comment]
> I found the solution just logout your account

Not logged in to begin with, so no account to logout of!

**Comment by GH1858** [comment]
> > > Perhaps Google has finally blocked their YouTube API feeding InnerTune?
> > 
> > I don't know, but if they blocked their YouTube API, then I have to switch to OuterTune.
> 
> I found the solution just logout your account

Nope, still the same error.

**Comment by GH1858** [comment]
Still the same.

**Comment by AkemiKami201** [comment]
> > > Perhaps Google has finally blocked their YouTube API feeding InnerTune?
> > 
> > I don't know, but if they blocked their YouTube API, then I have to switch to OuterTune.
> 
> I found the solution just logout your account

No, still the same

**Comment by vanshbrhe** [comment]
Same happened please fix it fast I waana listen new songs

**Comment by aayush1693** [comment]
seems like no fix yet..

**Comment by gechoto** [comment]
If you want to help you can test this build and report if it works:
https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291

- You need to be logged in to download
- Please test with a fresh install (uninstall previous debug builds before and do NOT restore any backup)

**Comment by GH1858** [comment]
> If you want to help you can test this build and report if it works:
> https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291

I tried to test the build but it's said 404 Error.

**Comment by gechoto** [comment]
> I tried to test the build but it's said 404 Error.

You need to be logged in to download

**Comment by GH1858** [comment]
> 
> > I tried to test the build but it's said 404 Error.
> 
> You need to be logged in to download

Is there any malware or viruses? Sorry this was first time using it.

**Comment by gechoto** [comment]
> Is there any malware or viruses? Sorry this was first time using it.

The APK is generated by GitHub so if you trust the website you are already on it will be fine.
My changes are also public so you can have a look at the code and see there is nothing bad in it.

**Comment by GH1858** [comment]
Okay thanks.

**Comment by kr812** [comment]
> > If you want to help you can test this build and report if it works:
> > https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291
> 
> I tried to test the build but it's said 404 Error.

Bro i think you did it because it's working did you changed the Api

**Comment by GH1858** [comment]
Yeah I just now log in and it worked.

**Comment by kr812** [comment]
Means you didn't change the 

> If you want to help you can test this build and report if it works: https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291
> 
> * You need to be logged in to download
> * Please test with a fresh install (uninstall previous debug builds before and do NOT restore any backup)

Hlo bro did you changed the Api of this app

**Comment by GH1858** [comment]
No

**Comment by kr812** [comment]
> No

Then I try to build apk it's give me error

**Comment by kr812** [comment]
> No

![Uploading Screenshot_2024-12-30-00-36-15-699_com.android.chrome.jpg…]()

**Comment by aayush1693** [comment]
it works guys!

**Comment by techy2245** [comment]
even in the latest debug there is error issue and some songs dont play and doesnt even play

**Comment by HGStyle** [comment]
> If you want to help you can test this build and report if it works: https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291
> 
> * You need to be logged in to download
> * Please test with a fresh install (uninstall previous debug builds before and do NOT restore any backup)

It works great for me, thanks!

![IMG_20241231_164215](https://github.com/user-attachments/assets/d452f2b3-14dc-4604-90d1-90de881f962d)

**Comment by HGStyle** [comment]
Just saw it only worked for songs frol YT Music and not from YT, ie songs with square thumbnail works but not those with rectangle thumbnail.

**Comment by vvero2020** [comment]
> If you want to help you can test this build and report if it works: https://github.com/z-huang/InnerTune/actions/runs/12536995462/artifacts/2369536291
> 
> * You need to be logged in to download
> * Please test with a fresh install (uninstall previous debug builds before and do NOT restore any backup)

it seems to work very well
no logged
no proxy

try

**Comment by SpiritStack** [comment]
This has stopped working now

**Comment by HGStyle** [comment]
> This has stopped working now

This seems to be backward compatible with InnerTune backups and still works flawlessly: https://github.com/DD3Boh/OuterTune

**Comment by gechoto** [comment]
> This has stopped working now

please test with the latest version:
https://github.com/z-huang/InnerTune/actions/runs/12946584020/artifacts/2479712770
